cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js

```diff
@@ -1,26 +1,18 @@
 import {
-  getCookieBanner,
-  getCookieBannerAccept,
-  getCookieBannerReject,
+  getCookieBannerAmp,
+  getCookieBannerAcceptAmp,
   getPrivacyBanner,
   getPrivacyBannerAccept,
+  getCookieBannerManageSettings,
+  getCookieBannerManageSettingsButton,
+  getCookieBannerAcceptInManageSettings,
+  getCookieBannerRejectInManageSettings,
 } from '../utilities/cookiePrivacyBanner';
 import visitPage from '../../../support/helpers/visitPage';
 
 export default ({ service, variant, pageType, path }) => {
   it('should have a privacy & cookie banner, which disappears once "accepted" ', () => {
     getPrivacyBanner(service, variant).should('be.visible');
```

Comment:
Is there a reason that the test  it('should have a privacy & cookie banner, which disappears once "accepted" ' now only checks the visibility of the privacy banner? If this is a discussion I missed, and we do want the rest of the steps removed, we should change the name of the test because it no longer accepts the banner.

@paruchurisilpa @andrewscfc

Reply:
I'm not sure why those steps have been removed, that doesn't seem correct actually good spot